src/views/jobManager/jobManagerView.ts

```diff
@@ -303,6 +304,12 @@ export class JobManagerView implements TreeDataProvider<any> {
     updateStatusBar();
 
     const selectedJob = JobManager.getSelection();
+    const selectedSchema = selectedJob.job.options.libraries[0];
```

Comment:
We need to add a check here to make sure `selectedJob` is not `undefined`, like the lines below.